src/modules/27-platform/connectors/components/CreateConnector/GcpConnector/StepAuth/GcpAuthentication.tsx

```diff
@@ -70,13 +72,27 @@ const GcpAuthentication: React.FC<StepProps<StepConfigureProps> & GcpAuthenticat
   const DelegateCards: DelegateCardInterface[] = [
     {
       type: DelegateTypes.DELEGATE_OUT_CLUSTER,
-      info: getString('platform.connectors.GCP.delegateOutClusterInfo')
+      info: getString('platform.connectors.GCP.delegateOutClusterInfo'),
+      icon: 'service-kubernetes'
     },
     {
       type: DelegateTypes.DELEGATE_IN_CLUSTER,
-      info: getString('platform.connectors.GCP.delegateInClusterInfo')
+      info: getString('platform.connectors.GCP.delegateInClusterInfo'),
+      icon: 'delegates-blue'
     }
   ]
+  const oidcAddedDelegateCards = (): DelegateCardInterface[] => {
+    return PL_GCP_OIDC_AUTHENTICATION
+      ? [
+          ...DelegateCards,
+          {
+            type: DelegateTypes.DELEGATE_OIDC,
+            info: 'OIDC',
+            icon: 'oidc-authentication'
+          }
+        ]
+      : DelegateCards
+  }
```

Comment:
This can be an array, rather than a function.

```suggestion
  const delegateCardsWithOIDC: DelegateCardInterface[] = [
          ...DelegateCards,
          ...(PL_GCP_OIDC_AUTHENTICATION ? [{
            type: DelegateTypes.DELEGATE_OIDC,
            info: 'OIDC',
            icon: 'oidc-authentication'
          }]:[])
 
  
```